src/content/api/cli.mdx

```diff
@@ -608,3 +608,18 @@ To use `webpack v5.32.0`:
 ```bash
 WEBPACK_PACKAGE=webpack-5 npx webpack
 ```
+
+## Troubleshooting
+
+### TypeError [ERR_UNKNOWN_FILE_EXTENSION]: Unknown file extension ".ts" for ./webpack.config.ts
+
+You might encounter this error in the case of typescript ECMA modules (i.e. `type: "module"` in `package.json`).
+
+`webpack-cli` supports `commonjs` and `ES` format, at first it tries to load a configuration using `require()`, if it fails with an error code of `'ERR_REQUIRE_ESM'` (special code for this case) it tries to load the configuration using `import()`.
```

Comment:
```suggestion
`webpack-cli` supports configuration in both `CommonJS` and `ESM` format, at first it tries to load a configuration using `require()`, once it fails with an error code of `'ERR_REQUIRE_ESM'` (a special code for this case) it would try to load the configuration using `import()`.
```